.travis.yml

```diff
@@ -40,6 +40,10 @@ install:
   - cp travisci/MultiTestDB.conf.travisci.SQLite modules/t/MultiTestDB.conf.SQLite
   - mysql -u root -h localhost -e 'GRANT ALL PRIVILEGES ON *.* TO "travis"@"%"'
 
+before_script:
+  # This should probably be removed on release branches
```

Comment:
I'll be laying awake for an extra 30 seconds tonight worrying about how this opens us up to a risk of false negatives in testing. Having removal of this line in the release checklist mitigates this risk. It would be nice if there were some way to have harness.sh check for a release mismatch in the test databases from variation, etc. and conclude that we are on an n+1 branch, but at the moment I'm not sure about how to do that sensibly and not be at least as much of a pain as remembering to delete this line.

Reply:
I have already noted the need to disable creation of this file on release branches in Core Release Procedure, just in case. As for automation, one could always consider the value of $TRAVIS_BRANCH to decide whether to disable version checking or not - but it should be done carefully because we've got development/PR branches to account for in addition to _master_ and release branches.

Reply:
Sorry, I wasn't clear -- should have said that I saw you added that item to the release checklist, which takes care of the issue. Thinking about the issue a bit more, I'm not sure, given how many other test DBs this needs to interact with, that a systematic approach would ever be possible.